### DISCUSSION TITLE
How disable NAME and EMAIL edit in user profile? Allow only PASSWORD change.

### DISCUSSION THREAD

**Discussion by alexworldbr** [Jacobtims/filament-breezy#227] [Q&A]
Hi,

How disable NAME and EMAIL edit in default user profile for allow only PASSWORD change?

Att,

AlexDevBr

**Comment by alexworldbr** [comment]
Hi @jeffgreco13 ,

Can you see this? 

Tks

**Reply by joaopaulolndev** [reply]
You need to create a custom profile
